package.json

```diff
@@ -753,10 +779,18 @@
     "azure-iot-device": "^1.8.1",
     "azure-iot-device-mqtt": "^1.8.1",
     "azure-iothub": "^1.8.1",
+    "body-parser": "^1.18.2",
+    "dotenv": "^5.0.1",
+    "dummy-json": "^2.0.0",
+    "express": "^4.16.3",
     "fs-extra": "^7.0.0",
+    "is-port-reachable": "^2.0.0",
```

Comment:
Are is-port-reachable, request, request-promise. semver, used?
Please also double check for other.

Reply:
Removed. Also removed `sinon`, `@types/dotenv` and other 3 unused dependencies.